src/Functions/FunctionsStringHashFixedString.cpp

```diff
@@ -21,6 +21,11 @@
 #    include <openssl/sha.h>
 #endif
 
+#if defined USE_ETHASH && USE_ETHASH
```

Comment:
This duplication is strange.
Do not use `defined`, so the compiler will check if you included the config.